plugins/plugin-klogs/pkg/instance/logs.go

```diff
@@ -10,9 +10,28 @@ import (
 	"go.uber.org/zap"
 )
 
+func (i *instance) enrichAutolinks(fields []Field) []Field {
+	autolinkIntegrations := make([]IntegrationAutolinks, 0)
```

Comment:
I don't expect this slice to become huge, same with `i.integrations`. If desired we can filter the `autolinkIntegrations` slice outside of this method (which is invoked on every request). Furthermore I could make it a `map[string]IntegrationAutolinks` with the `columnName` as key. This would further reduce the overhead in L22-L26.